joint_limits:
    joint1:
      has_position_limits: true
      min_position: 0.0
      max_position: 1.0
      has_velocity_limits: true
      max_velocity: 1.0472
      has_acceleration_limits: true
      max_acceleration: 5.0
      has_jerk_limits: true
      max_jerk: 100.0
      has_effort_limits: true
      max_effort: 255
      min_effort: -255